### ISSUE TITLE
error on running db/migrate.cr- Can't DROP 'total'; check that column/key exists (Exception)

### ISSUE THREAD

**Issue by azah** [drujensen/kemalyst-model#19]
This field was never in my model to begin with.

Looks like it's an issue with #prune.   
Even if I add a field in mysql called 'total', the error still occurs (and the new 'total' field is also dropped).

stacktrace:
```
[4766327] *CallStack::unwind:Array(Pointer(Void)) +87
[4766218] *CallStack#initialize:Array(Pointer(Void)) +10
[4766170] *CallStack::new:CallStack +42
[4688648] *raise<Exception>:NoReturn +24
[4688622] ???
[5091984] *MySql::Connection#handle_err_packet<MySql::ReadPacket>:NoReturn +80
[5498273] *MySql::Statement#perform_exec_or_query<Slice(Float32 | Float64 | Int32 | Int64 | Slice(UInt8) | String | Nil)>:(DB::ExecResult | MySql::ResultSet) +5857
[5492159] *MySql::Statement#perform_exec<Slice(Float32 | Float64 | Int32 | Int64 | Slice(UInt8) | String | Nil)>:DB::ExecResult +63
[5502771] *DB::Statement+ +51
[5502684] *DB::Statement+ +60
[5484443] *DB::PoolStatement#exec:DB::ExecResult +203
[5068117] *DB::Database +37
[5048973] *Kemalyst::Adapter::Mysql#prune<String, Hash(String, String)>:Array(String) +765
[5054277] *Post +53
[4646396] ???
[4707641] main +41
[140029813202757] __libc_start_main +245
[4641177] ???
[0] ???
```

**Comment by drujensen** [comment]
ok, I will take a look when I get a chance

**Comment by drujensen** [comment]
Released v0.2.5 to address this issue.  @azah Thanks for reporting this!
